api/resid/gvk.go

```diff
@@ -207,5 +208,7 @@ func (x Gvk) toKyamlTypeMeta() yaml.TypeMeta {
 // IsNamespaceableKind returns true if x is a namespaceable Gvk
 // Implements https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#not-all-objects-are-in-a-namespace
 func (x Gvk) IsNamespaceableKind() bool {
-	return x.toKyamlTypeMeta().IsNamespaceable()
+	openapi.ResetOpenAPI()
```

Comment:
Why do we reset the openapi here? I don't think that is needed.

Reply:
fixed